gokart/conflict_prevention_lock/task_lock_wrappers.py

```diff
@@ -6,38 +6,6 @@
 logger = getLogger(__name__)
 
 
-def _wrap_with_lock(func, task_lock_params: TaskLockParams):
-    if not task_lock_params.should_task_lock:
-        return func
-
-    def wrapper(*args, **kwargs):
-        task_lock = set_task_lock(task_lock_params=task_lock_params)
-        scheduler = set_lock_scheduler(task_lock=task_lock, task_lock_params=task_lock_params)
-
-        try:
-            logger.debug(f'Task lock of {task_lock_params.redis_key} locked.')
-            result = func(*args, **kwargs)
-            task_lock.release()
-            logger.debug(f'Task lock of {task_lock_params.redis_key} released.')
-            scheduler.shutdown()
-            return result
-        except BaseException as e:
-            logger.debug(f'Task lock of {task_lock_params.redis_key} released with BaseException.')
-            task_lock.release()
-            scheduler.shutdown()
-            raise e
-
-    return wrapper
-
-
-def wrap_with_run_lock(func: Callable, task_lock_params: TaskLockParams):
```

Comment:
[qeustion] Any reason why did you move this logic to wrap_run_with_lock.py ?  (and rename `with_run_lock` to `run_with_lock` )

its logic is similar with this file line 65-79, so put them together seems fine ?

(move `wrap_run_with_lock` after line 80 maybe better because similar codes become closer)

Reply:
@yokomotod I've moved the codes!